### ISSUE TITLE
[Slider] Support custom increment/decrement levels

### ISSUE THREAD

**Issue by codeman7** [material-components/material-components-ios#4507]
Currently we don't allow clients to have custom increment/decrement levels within VoiceOver

## Expected behavior

A client should be able to set these values through a property.

## Actual behavior

We currently have this as a constant.
